### PR TITLE
7357 Added empty comment object

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#7357

*Description of changes:*
- added empty comment object
- updated browsers list
- updated Storybook
- fixed small lint issues

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/DatePicker/DatePicker.vue [hunk 1]
```diff
@@ -48,7 +48,6 @@ import { FormIF } from '@/interfaces'
 
 @Component({})
 export default class DatePicker extends Vue {
-
```
Comment (severinbeauvais) [inline src/components/DatePicker/DatePicker.vue:51]: This was a lint warning.

File: tests/unit/Certify.spec.ts [hunk 2]
```diff
@@ -241,7 +241,8 @@ describe('Certify', () => {
   })
 
   it('disables input field when prop is passed true', () => {
-    const wrapper: Wrapper<Certify> = createComponent(null, true, false, null, false, false, [], true, 'mockBusinessEmail', 'mockPartyEmail', true)
```
Comment (severinbeauvais) [inline tests/unit/Certify.spec.ts:244]: This was a lint warning (long line).

File: src/interfaces/comment-interface.ts [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
-  submitterDisplayName: string;
-  timestamp: string;
+  businessId?: string
+  comment: string
+  filingId?: string
+  submitterDisplayName: string
+  timestamp: string
+}
+
+export const EmptyComment: CommentIF = {
```
Comment (severinbeauvais) [inline src/interfaces/comment-interface.ts:10]: It's nice to have this kind of object to initialize things (or provide prop defaults).

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

**Review by andrepestana-aot — APPROVED** [review]

**Pull request merged**